minikube-gradle-plugin/build.gradle

```diff
@@ -16,6 +16,7 @@ repositories {
 
 dependencies {
   compile gradleApi()
+  compile 'com.google.guava:guava:23.5-jre'
```

Comment:
I think I'd rather not add this dependency if we can avoid it. We're trying to keep dependencies to a minimum as buildscript classpath level dependencies tend to have weird behavior when they clash in gradle. Looks like we can just replace the preconditions check in MinikubeExtension with simple custom code that throws NullPointerException?